.gitignore

```diff
@@ -3,3 +3,5 @@
 /node_modules
 /out
 /.vscode/
+composer.lock
```

Comment:
@NielsdeBlaauw and co. Thanks for the new and autoloading functionality and other features ;)

Just curious. Why is the composer.lock unused? You don't mind having the possiblity of having different installation per developer working on the same project? And having the possiblity that a staging enviroment could differ from a production environment?

Reply:
Versions are still locked in the theme's lock file. Clarkson just doesn't lock specific version itself anymore. 

**In this specific case:** the tests running on php7.2 might want different versions of certain dependencies then the same test running on php7.4.